mtgjson4/compile_mtg.py

```diff
@@ -544,6 +586,15 @@ def build_mtgjson_card(
         ):
             del mtgjson_card["names"]
 
+    # Since we built meld cards later, we will add the "side" attribute now
+    if len(mtgjson_card.get("names", [])) == 3:  # MELD
+        if mtgjson_card["name"] == mtgjson_card["names"][0]:
+            mtgjson_card["side"] = "a"
+        elif mtgjson_card["name"] == mtgjson_card["names"][2]:
```

Comment:
@ZeldaZach Should this be `[1]`? Also we've noticed that there's inconsistency in the name order:

```
[11] pry(main)> j['EMN']['cards'].find { |c| c['name'] =~ /Brisela/ }['side']
=> "c"
[12] pry(main)> j['EMN']['cards'].find { |c| c['name'] =~ /Brisela/ }['names']
=> ["Bruna, the Fading Light", "Brisela, Voice of Nightmares", "Gisela, the Broken Blade"]
[13] pry(main)> j['EMN']['cards'].find { |c| c['name'] =~ /Rats/ }['names']
=> ["Graf Rats", "Midnight Scavengers", "Chittering Host"]
[14] pry(main)> j['EMN']['cards'].find { |c| c['name'] =~ /Chittering/ }['names']
=> ["Graf Rats", "Midnight Scavengers", "Chittering Host"]
[15] pry(main)> 
```

Reply:
Brisela is `[1]` but Chittering Host is `[2]` in the other

Reply:
Ah, according to the docs:

> Names of each face on the card. Meld cards are listed in the order of CardA, Meld, CardB.

So maybe there's an issue with ordering here